assets/pages/regular-contributions/regularContributions.jsx

```diff
@@ -7,100 +7,62 @@ import { applyMiddleware, compose } from 'redux';
 const countryGroup = detectCountryGroup();
-const country = detectCountry();
-const currency = detectCurrency(countryGroup);
+const { amount } = parseAmount(getQueryParameter('contributionValue'), contributionType, countryGroup);
 
-const contributionType = parseContrib(getQueryParameter('contribType'), 'MONTHLY');
-const contributionAmount = parseAmount(getQueryParameter('contributionValue'), contributionType, countryGroup).amount;
+function getPaymentMethod(): ?PaymentMethod {
+  const pm: ?string = storage.getSession('paymentMethod');
+  if (pm && (pm === 'DirectDebit' || pm === 'Stripe' || pm === 'PayPal')) {
+    return (pm: PaymentMethod);
+  }
+  return null;
+}
 
-const title = {
-  annual: ['Make an annual', 'contribution'],
-  monthly: ['Make a monthly', 'contribution'],
-};
 
 /* eslint-disable no-underscore-dangle */
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 /* eslint-enable */
 
 const store = pageInit(
-  reducer(contributionAmount, currency),
+  reducer(amount, getPaymentMethod()),
   undefined,
   composeEnhancers(applyMiddleware(thunkMiddleware)),
 );
 
 user.init(store.dispatch);
 store.dispatch(setPayPalButton(window.guardian.payPalType));
 
-const state: PageState = store.getState();
-
-
-// ----- Render ----- //
-
-const content = (
-  <Provider store={store}>
-    <div className="gu-content">
-      <TestUserBanner />
-      <SimpleHeader />
-      <CirclesIntroduction headings={title[contributionType.toLowerCase()]} />
-      <hr className="regular-contrib__multiline" />
-      <div className="regular-contrib gu-content-margin">
-        <InfoSection heading={`Your ${contributionType.toLowerCase()} contribution`} className="regular-contrib__your-contrib">
-          <PaymentAmount
-            amount={state.page.regularContrib.amount}
-            currency={state.page.regularContrib.currency}
-          />
-          <Secure />
-        </InfoSection>
-        <InfoSection heading="Your details" headingContent={<Signout />} className="regular-contrib__your-details">
-          <DisplayName />
-          <FormFields />
-        </InfoSection>
-        <InfoSection heading="Payment methods" className="regular-contrib__payment-methods">
-          <RegularContributionsPayment contributionType={contributionType} />
-        </InfoSection>
-      </div>
-      <div className="terms-privacy gu-content-filler">
-        <InfoSection className="terms-privacy__content gu-content-filler__inner">
-          <TermsPrivacy country={country} />
-          <ContribLegal />
-        </InfoSection>
+const router = (
+  <BrowserRouter>
+    <Provider store={store}>
+      <div>
+        <Route exact path="/contribute/recurring" component={RegularContributionsPage} />
+        <Route exact path="/contribute/recurring/thankyou" component={ContributionsThankYouPage} />
+        <Route exact path="/contribute/recurring/pending" component={ContributionsThankYouPage} />
```

Comment:
`routes.recurringContribCheckout`, `routes.recurringContribThankyou` and `routes.recurringContribPending`.